documentation/API_documentation/QoD_Stable_Bandwidth_API.md

```diff
@@ -0,0 +1,180 @@
+
+Before starting with the API, the developer needs to provide base-URL, security credentials and App-Flow parameters with specified QoS profile.
+
+**Base-URL**
+The RESTful Stable Throughput API endpoint, for example [**https://telekom-api.developer.telekom.com/5g-latency**](https://telekom-api.developer.telekom.com/5g-throughput)
+
+**Authentication**
+Configure security access keys such as O-Auth2 client credentials to be used by Client applications which will invoke the QoD API.
+
+**QoS Profile**
+Define throughput requirements of the application and identify QoS profile class which maps into the required performance category.
+
+**App-Flow**
+Describes the precise flow the developer wants to prioritize and have stable bandwidth for. This flow is described using source and destination IP addresses, ports and protocols with flow direction.
+
+**Duration**
+Define the number of seconds for which the QoD session should be created. This parameter is optional and if not specified, the session is either deleted on user request or if default expiration limit of 24hrs has been reached.
+
+**Notification URL and token**
+Developers have a chance to specify callback URL on which notifications (eg. session termination) regarding the session can be received from the service provider. This is also an optional parameter.
+
+Sample API invocations are presented in Section 4.5.
+
+## 3\. Authentication and Authorization
+
+The QoD Service API makes use of the client credentials grant which is applicable for server to server use cases involving trusted partners or clients without any protected user data involved.
+In this method the API invoker client is registered as a confidential client with an authorization grant type of client\_credentials [3].
+
+## 4\. API Documentation
+
+### 4.1 Details
+The usage of the QoD throughput API is based on QoS profile classes and parameters which define App-Flows.
+Based on the API, QoS sessions can be created, queried, and deleted. Once a proper QoS profile throughput class is requested, application users get a prioritized service with stable throughput even in the case of congestion.The QoD bandwidth API has the following characteristics:
+* A specified App-Flow is prioritzed to ensure stable throughput for that flow
+* The prioritized App-Flow is described by providing additional information such as protocols, ports, uplink/downlink direction of flow etc. 
+* Stable bandwidth is requested by selecting from the list of QoS profiles made available by the service provider (e.g. Small ,Medium, Large) to map throughput requirements
+* If the developer can optionally specify the duration for which he needs the prioritized App-flow
+* The developer can optionally also specify callback URL on which notifications for the session can be sent
+<br>
+Following diagram shows the interaction between different components 
+
+<br />
+<img src="./resources/QoD_bandwidth_details.PNG" alt="QoD_BM" title="QoD Bandwidth Management"> 
```

Comment:
+: Register for Notification call back.
Same comment for QoD Latency.

Reply:
Registering notification call back is part of `POST /sessions` ... which change do you request here?